### ISSUE TITLE
[DocDB] The test RaftConsensusITest.HintedLeader is flaky

### ISSUE THREAD

**Issue by mbautin** [yugabyte/yugabyte-db#10395]
Jira Link: [DB-897](https://yugabyte.atlassian.net/browse/DB-897)
### Description

Log:

https://gist.githubusercontent.com/mbautin/632d51005ad2562b9f50a5b1fea63db9/raw

The master leader sends a leader election request to a node with a hinted leader. But that request fails because the two other replicas are still bootstrapping, and a different peer becomes leader. The test fails.

**Comment by rthallamko3** [comment]
https://detective-gcp.dev.yugabyte.com/stability/test?branch=master&build_type=all&class=RaftConsensusITest&fail_tag=all&name=HintedLeader&platform=linux 


```

09/21/2022 18:18 | acb55a5fd~..acb55a5fd | 1 | 0
-- | -- | -- | --
09/20/2022 16:33 | a3984847d~..a3984847d | 1 | 0
09/17/2022 01:48 | dec080592~..dec080592 | 1 | 0

09/21/2022 18:18	[acb55a5fd~..acb55a5fd](https://github.com/yugabyte/yugabyte-db/compare/acb55a5fd98f8a28caa5c3ee4209cd009c26ea88~...acb55a5fd98f8a28caa5c3ee4209cd009c26ea88)	1	0
09/20/2022 16:33	[a3984847d~..a3984847d](https://github.com/yugabyte/yugabyte-db/compare/a3984847d77c07474fcc924cb04b68cc4abfa48f~...a3984847d77c07474fcc924cb04b68cc4abfa48f)	1	0
09/17/2022 01:48	[dec080592~..dec080592](https://github.com/yugabyte/yugabyte-db/compare/dec0805927f6b77f0e6b8918ef69b6efa5a9ba30~...dec0805927f6b77f0e6b8918ef69b6efa5a9ba30)	1	0

```

**Comment by rthallamko3** [comment]
@Huqicheng , Can you take a look as part of the hinted leader change that you are looking at in the context of tablet-splitting item?

**Comment by Huqicheng** [comment]
@rthallamko3 This issue was fixed by feec0ad4525b5cceb8f015d1911027903e398b32
